### DISCUSSION TITLE
wrong format out in error

### DISCUSSION THREAD

**Discussion by gypais** [blakeblackshear/frigate#12731] [General Support]
### Describe the problem you are having

When you make a mistake in the duration of an export, this causes an error, which is normal, but the output seems badly formatted:

wrong format out in error : 
Type
error
Timestamp
2024-08-02 21:44:42
Tag
frigate.record.export
Message
b'Invalid duration specification for ss: 00:-28701068:8\n'

b and \n ...

### Steps to reproduce

NA

### Version

System 0.14.0-7c39b17

### Frigate config file

```yaml
NA
```

### Relevant log output

```shell
Type
error
Timestamp
2024-08-02 21:44:42
Tag
frigate.record.export
Message
b'Invalid duration specification for ss: 00:-28701068:8\n'
```

### Operating system

Other Linux

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

pg x6 weq 2 + cloudcam

**Comment by NickM-27** [comment]
This error comes directly from ffmpeg, this is not something we control
